packages/gatsby-theme-carbon/src/components/LeftNav/LeftNavItem.js

```diff
@@ -24,17 +24,14 @@ const LeftNavItem = props => {
         const pathname = pathPrefix
           ? location.pathname.replace(pathPrefix, '')
           : location.pathname;
-        const isActive = items.some(
-          item => pathname === item.path || pathname === `${item.path}/`
+
```

Comment:
Ah! That totally makes sense, thanks again for your patience with this, just wanted to add a comment for if we have to revisit this logic down the line.

```suggestion
// If item a dropdown, ignore the last path segment to account for tabbed
// items, which won't show up in the items array. For non-dropdowns,
// just check for exact path match while removing trailing slash.
```